dub.sdl

```diff
@@ -29,7 +29,8 @@ subPackage {
     "src/dmd/tokens.d" \
     "src/dmd/utf.d"
 
-  preGenerateCommands `cd "$${DUB_PACKAGE_DIR}" && ./config.sh generated/dub VERSION /etc`
+  preGenerateCommands `cd "$${DUB_PACKAGE_DIR}" && $${DC} -run ./config.d generated/dub VERSION /etc` platform="posix"
```

Comment:
DAutoTest fails with:

```
.generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dub build --compiler=.generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dmd --root=ddoc && \
	mv ddoc/ddoc_preprocessor .generated/ddoc_preprocessor
Fetching libdparse 0.8.7 (getting selected version)...
Fetching stdx-allocator 2.77.2 (getting selected version)...
Invalid source/import path: /dev/shm/dtest/work/repo/dmd/generated/dub
Running pre-generate commands for dmd:frontend...
/bin/sh: .generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dmd: No such file or directory
Command failed with exit code 127: cd "${DUB_PACKAGE_DIR}" && ${DC} -run ./config.d generated/dub VERSION /etc
posix.mak:922: recipe for target '.generated/ddoc_preprocessor' failed
```

So it looks like you need to save PWD

Reply:
(did this for you)

Reply:
👍.